spec/controllers/my_modules_controller_spec.rb

```diff
@@ -153,23 +153,24 @@
       end
     end
 
-    context 'when status not found' do
+    context 'when status not exist' do
```

Comment:
Hm... I think a better approach to test validation is with the model spec. It is not obvious why you get 422 here.